lib/usd/ui/layerEditor/mayaCommandHook.cpp

```diff
@@ -54,7 +59,40 @@ MString executeMel(const std::string& commandString)
 // maya doesn't support spaces in undo chunk names...
 MString cleanChunkName(QString name) { return quote(name.replace(" ", "_").toStdString()).c_str(); }
 
+std::string GetProxyShapeName(const std::string& proxyShapePath)
```

Comment:
This function and the one below should be "lowerCamelCase" so "get" not "Get"

Reply:
Done!